policyengine_us/variables/gov/states/ok/tax/income/ok_agi_subtractions.py

```diff
@@ -18,4 +18,7 @@ def formula(tax_unit, period, parameters):
         pensions = tax_unit.members("taxable_pension_income", period)
         p = parameters(period).gov.states.ok.tax.income.agi.subtractions
         capped_pension = tax_unit.sum(min_(p.pension_limit, pensions))
-        return oasdi + capped_pension
+        military_retirement_exclusion = tax_unit(
+            "ok_military_retirement_exclusion", period
+        )
+        return oasdi + capped_pension + military_retirement_exclusion
```

Comment:
please make this a list parameter, and put the capped pension subtraction into a new variable